workflows/templates/post-rebuild-worker.yaml

```
@@ -62,8 +61,7 @@ spec:
               - name: scriptContent
                 value: |
                   ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null {{inputs.parameters.targetNcn}} \
-                    -t "SW_ADMIN_PASSWORD='{{inputs.parameters.switchPassword}}' \
-                        GOSS_BASE=/opt/cray/tests/install/ncn \
+                    -t  "GOSS_BASE=/opt/cray/tests/install/ncn \
```

Extra space was added
```suggestion
                    -t "GOSS_BASE=/opt/cray/tests/install/ncn \
```